crates/kilt-dip-primitives/src/utils.rs

```diff
@@ -80,3 +81,26 @@ where
 		Self(sp_std::vec![sp_std::vec![T::default(); 128]; 64])
 	}
 }
+
+pub(crate) fn calculate_parachain_head_storage_key(para_id: u32) -> StorageKey {
+	StorageKey(
+		[
+			frame_support::storage::storage_prefix(b"Paras", b"Heads").as_slice(),
```

Comment:
would it make sence to have the values "paras" and "Heads" into a constant?

Reply:
It's only ever used in this function, I don't think it makes that much sense. If we start using it in other places as well, will definitely move them out to constants.

Reply:
Okay, it's probably personal preferences. Will mark it as resolved